src/main/java/com/photolocos/enterprise/service/IPhotoService.java

```diff
@@ -8,15 +8,15 @@
 
 public interface IPhotoService {
 
-    PhotoDTO fetchByLocation(String location);
+    Set<PhotoDTO> fetchByArea(String area) throws Exception;
 
-    Set<PhotoDTO> fetchByTag(String tag);
+    Set<PhotoDTO> fetchByTag(String tag) throws Exception;
 
     PhotoDTO savePhoto(PhotoDTO photo, MultipartFile image);
 
-    Set<PhotoDTO> fetchAll();
+    Set<PhotoDTO> fetchAll() throws Exception;
 
-    Set<PhotoDTO> fetchPhotoByCity(String city);
+    Set<PhotoDTO> fetchPhotoByStateAndCity(String state, String city) throws Exception;
 
-    Set<LocationDTO> fetchLocationByCity(String city);
+    LocationDTO fetchLocationByStateAndCity(String state, String city) throws Exception;
```

Comment:
Based on the method call from the controller and the parameters, this should be fetchByStateAndCity